src/main/java/com/github/group3coursework/Reports/PeopleWhoSpeakDiffLang.java

```diff
@@ -0,0 +1,59 @@
+package com.github.group3coursework.Reports;
```

Comment:
Not major, but maybe this file could be named better.

Reply:
Do you mean the package? Or the 'PeopleWhoSpeakDiffLang'

Reply:
Yeah the 'PeopleWhoSpeakDiffLang'. I don't know how to do a full file comment :/

Reply:
Aye you're right, name's confusing